src/System.Net.Http/tests/FunctionalTests/CancellationTest.cs

```diff
@@ -2,161 +2,442 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
+using System.Collections.Generic;
 using System.Diagnostics;
+using System.Linq;
 using System.IO;
 using System.Net.Test.Common;
 using System.Threading;
 using System.Threading.Tasks;
-
 using Xunit;
-using Xunit.Abstractions;
 
 namespace System.Net.Http.Functional.Tests
 {
-    public class CancellationTest : HttpClientTestBase
+    public class HttpClientHandler_CancellationTest : HttpClientTestBase
```

Comment:
Filename?  (I don't particularly care either way, just trying to understand what the pattern is for tests generally)

Reply:
> just trying to understand what the pattern is for tests generally

We're all over the place.  I can change the file name.